README.md

```diff
@@ -1,13 +1,16 @@
 # Project Name
 
-Replace this readme with your own information about your project.
+Here is my basic website design for a blog post. I detail designing a bedroom for my daughter.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The brief was to design a basic webpage, including images.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+My approach was to use a design board I had previously designed, to illustrate how I designed a bedrrom for my 8-year old daughter. 
+I added internet links to sources for the items. I have used Adobe Illustrator jpg files, added a menu section and some styling. 
+I would like to usee CSS Grid next time to style (I feel my website is a bit too basic and amateur-ish)
```

Comment:
Nice with the design board and menu! 👍 
If you want to use CSS Grid next time, I recommend: https://css-tricks.com/snippets/css/complete-guide-grid/